gcp-serverless/index.js

```diff
@@ -3,16 +3,28 @@ const config = require('./config.json')
 const { getColorCodes } = require('./getColorCodes')
 const callParticleApi = require('./callParticleApi')
 
-exports.callparticle = (request, response) => {
-  // @todo, does this var need to be escaped?
-  const deviceId = request.query.coreid
-  const particleToken = config.PARTICLE_TOKEN
-  const makeRequest = async () => {
-    const colorList = await getColorCodes(config.GITHUB_TOKEN)
-    await callParticleApi(particleToken, deviceId, colorList)
-    response.status(200).send(colorList)
+const makeRequest = async (coreid) => {
```

Comment:
`makeRequest` and `callparticle` seem like redundant function name. It's even more confusing with `callParticleApi` is a separate function.

Reply:
That's fair. At least this PR is not making the situation any worse. I could do a follow up that rethinks the naming here. I think I went overboard with separating files and and functions. I see this PR as an intermediate step in the right direction.

Reply:
👍